### ISSUE TITLE
Merging of pull requests created by GitHub-actions can't be done

### ISSUE THREAD

**Issue by jacobcook1995** [ImperialCollegeLondon/virtual_rainforest#71]
**Describe the bug**
Pull requests that are automatically created by GitHub actions cannot be merged, because some of the checks either do not run or fail to report back to the pull request (e.g. #70). This issue appears to be fairly common [see](https://stackoverflow.com/questions/52200096/github-pull-request-waiting-for-status-to-be-reported/69169858#69169858). 

**To Reproduce**

1. Automatic update of pre-commit hooks is carried out in the early hours of Monday morning, this automatically creates a pull request
2. If you go to this pull request you will see that the `pre-commit.ci - pr` task has passed, but the `test` tasks just show a message saying "Expected — Waiting for status to be reported"
3. Pull request cannot be merged because "Required statuses must pass before merging"

**Expected behavior**
These tests should pass, allowing the pull request to be merged into develop.

**Additional context**
This issue seems to resolve itself when new GitHub actions are triggered by users, so one possible solution is pushing an empty commit to the branch, or closing and reopening the pull request. Alternatively, stack overflow suggests that by switching from using the default `GITHUB_TOKEN` to using a personal access token for this action this will allow the action to trigger other GitHub actions, which should fix the bug. However, while I have a PAT I'm not sure if it's sensible to be using my personal access token for this task? Or whether it is more appropriate to generate one for the bot to use (if that's possible)?

**Comment by dalonsoa** [comment]
Actually, I've just realised that the pre-commit action is no longer needed: I enabled the `pre-commit-ci` bot that deals with that automatically and - as you can see in https://github.com/ImperialCollegeLondon/virtual_rainforest/pull/72 - runs all checks as any normal PR. This bot is enabled at Imperial organisation level - although it needs to be activated on each repo that requires it. Details of how to customise this action can be found in [pre-commit.ci](https://pre-commit.ci/)

In summary, what you point out is indeed a problem in general, but in our particular case it can be sorted out simply by removing the pre-commit workflow altogether (this file https://github.com/ImperialCollegeLondon/virtual_rainforest/blob/develop/.github/workflows/pre-commit_autoupdate.yml) and letting the pre-commit bot to do its job.
